addons/project/views/project_sharing_project_task_views.xml

```diff
@@ -59,7 +59,7 @@
                                         <t t-else=""><field name="name" widget="name_with_subtask_count"/></t>
                                     </strong>
                                     <span invisible="context.get('default_project_id', False)"><br/><field name="project_id" required="1"/></span>
-                                    <span t-if="record.allow_milestones.raw_value and record.milestone_id.raw_value" t-attf-class="{{record.has_late_and_unreached_milestone.raw_value ? 'text-danger' : ''}}">
+                                    <span t-if="record.allow_milestones.raw_value and record.milestone_id.raw_value" t-attf-class="{{record.has_late_and_unreached_milestone.raw_value and !record.is_closed.raw_value ? 'text-danger' : ''}}">
```

Comment:
This field `is_closed` is being deleted in another task. https://www.odoo.com/web#id=3213526&cids=1&menu_id=4720&action=333&active_id=4105&model=project.task&view_type=form
Instead, check that the state is "1_done" or "1_canceled" (or starts with 1_ :wink: )